server/pulp/plugins/util/metadata_writer.py

```diff
@@ -148,17 +150,28 @@ def _open_metadata_file_handle(self):
         else:
             self.metadata_file_handle = open(self.metadata_file_path, 'w')
 
-    def _write_file_header(self):
+    def _write_xml_header(self):
         """
-        Write any headers for the metadata file
+        Write the initial <?xml?> header tag into the file handle.
         """
-        pass
+        assert self.metadata_file_handle is not None
+        _LOG.debug('Writing XML header into metadata file: %s' % self.metadata_file_path)
+        # newline is added to the toxml return inconsistently across versions of python,
+        # so .strip() it and explicitly add the newline here
```

Comment:
Whoah, that's odd. Is it OK to just let the different versions do what they will?